doc/modules/impute.rst

```diff
@@ -75,6 +83,48 @@ string values or pandas categoricals when using the ``'most_frequent'`` or
 :class:`SimpleImputer` can be used in a Pipeline as a way to build a composite
 estimator that supports imputation. See :ref:`sphx_glr_auto_examples_plot_missing_values.py`.
 
+.. _knnimpute:
+
+Nearest neighbors imputation
+===============================
+
+The :class:`KNNImputer` class provides imputation for completing missing
+values using the k-Nearest Neighbors approach. Each sample's missing values
+are imputed using values from ``n_neighbors`` nearest neighbors found in the
+training set. Note that if a sample has more than one feature missing, then
```

Comment:
This sentence is a bit cryptic to me. Maybe say "If multiple features are missing, not the sets of neighbors used for imputation can be different" or something like that?